stdlib/FileWatching/test/pidfile.jl

```diff
@@ -364,6 +364,43 @@ end
     close(lockf2)
 end
 
+@testset "mkpidlock ignore lock from dead process" begin
+    # stale_age = 1 here means that the lock will be:
+    # - respected for at least 1 second
+    # - ignored after 1 second if the pid no longer exists
+    # - ignored after 25 seconds if the pid still exists
+    @testset "process that finished before this process enters lock" begin
+        @assert !ispath("pidfile-3")
+        # mkpidlock with no waiting, in another process
+        run(`$(Base.julia_cmd()) -e "using FileWatching; mkpidlock(\"pidfile-3\", wait=false)"`)
+        t = @elapsed mkpidlock("pidfile-3", wait=false, stale_age=1, poll_interval=1, refresh=0)
+        @test t ≈ 1 atol=1
+        rm("pidfile-3", force=true)
```

Comment:
I don't see what these test that "mkpidlock for child" didn't already test. The `mkpidlock` calls returns a handle that you must close when you are done with (though will also get automatically destroyed before the process exits if that fails)